okapi-core/src/main/java/org/folio/okapi/util/EventBusChecker.java

```diff
@@ -0,0 +1,51 @@
+package org.folio.okapi.util;
+
+import io.vertx.core.CompositeFuture;
+import io.vertx.core.Future;
+import io.vertx.core.Vertx;
+import io.vertx.core.eventbus.DeliveryOptions;
+import io.vertx.core.spi.cluster.ClusterManager;
+import java.util.LinkedList;
+import java.util.List;
+
+public class EventBusChecker {
+  private static final String EVENT_NODE_CHECK = "org.folio.okapi.node.check.";
+
+  /**
+   * Check that event bus is operational by sending a message to each party.
+   * @param vertx Vert.x handle
+   * @param clusterManager manager ; null if not in clustered mode
+   * @return async result
+   */
+  public static Future<Void> check(Vertx vertx, ClusterManager clusterManager) {
```

Comment:
To handle the race condition when the node list changes you may have a check3 method that runs check(Vertx, ClusterManager) up to three times if it fails.